fcsparser/api.py

```diff
@@ -259,7 +273,16 @@ def _check_assumptions(self):
         keys = text.keys()
 
         if '$NEXTDATA' in text and text['$NEXTDATA'] != 0:
-            raise ParserFeatureNotImplementedError('Not implemented $NEXTDATA is not 0')
+            if '$ENDDATA' in text:
+                nextdata = text['$NEXTDATA']
+                enddata = int(text['$ENDDATA'])
+                if nextdata != enddata+1:
+                    logging.error('This file has multiple datasets, $NEXTDATA must equal to $ENDDATA+1')
+                else:
+                    logging.info('This file has multiple datasets')
```

Comment:
Not all users would know what this means, so it's nice for the logging message to inform the users what options are available to them. (e.g., provide specific parameters to the parser to extract the other data blocks.)